src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssProperties = sourceString
+    .trim()
+    .split(';')
+    .filter((str) => str.trim())
+    .map((str) => {
+      return str
+        .trim()
+        .split(':')
+        .map((s) => s.trim());
+    });
+
+  return Object.fromEntries(cssProperties);
```

Comment:
The function assumes that there will always be a colon ':' in the string to split key and value pairs. It's a good practice to add error handling for cases where the CSS string might be malformed, missing colons, or has additional unexpected characters.